docs/farming-&-staking/staking/operators/register-operator.mdx

```diff
@@ -205,17 +213,8 @@ It's crucial to fully sync your node before registering as an operator. Please f
 :::
 
 
-<details>
-<summary>
-Prefer a video? Expand for our installation video using PolkadotJS interface.
-</summary>
-<div>
-<iframe width="560" height="315" src="https://www.youtube.com/embed/w2U3CUJfI2c?si=mb-BRykmlrc49PPf" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowFullScreen></iframe>
-</div>
-</details>
-
 #### Register an operator using Subspace Staking interface (recommended)
-1. Proceed to the [Subspace Staking portal](https://staking.subspace.tools) and connect your wallet.
+1. Proceed to the [Subspace Staking portal](https://explorer.subspace.network/gemini-3h/staking) and connect your wallet.
```

Comment:
```suggestion
1. Proceed to the [Autonomys Staking portal](https://explorer.subspace.network/gemini-3h/staking) and connect your wallet.
```

Reply:
Here too, would suggest referring to it as "Astral"